docs/source/guides/custom_types_and_type_inference.ipynb

```diff
@@ -272,7 +272,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "If you create a new DataTable after updating the `Categorical` function, you can see that the `not_upc` column is no longer identified as a `Categorical` column, but is rather set to the default `NaturalLanguage` LogicalType. This is because the letters in the first row of the `not_upc` column cause our inference function to return `False` for this column, while the default `Categorical` function will allow non-numeric values to be present."
+    "If you initialize Woodwork on a DataFrame after updating the `Categorical` function, you can see that the `not_upc` column is no longer identified as a `Categorical` column, but is rather set to the default `NaturalLanguage` LogicalType. This is because the letters in the first row of the `not_upc` column cause our inference function to return `False` for this column, while the default `Categorical` function will allow non-numeric values to be present."
```

Comment:
I don't think it's quite clear why Natural Language is the default to Categorical

Reply:
Added these sentences to the end of the paragraph to try to help clarify. Does this help?

"After updating the inference function, this column is no longer considered a match for the `Categorical` type, nor does the column match any other logical types. As a result, the LogicalType is set to `NaturalLanguage`, the default type used when no type matches are found."

Reply:
Yes, that helps to clarify. Thanks!